docs/build-on-linea/tooling/data-indexers/subquery.md

```diff
@@ -0,0 +1,26 @@
+# Intro
```

Comment:
The `#` won't be necessary once we define a title in the front matter, as in my previous comment — so can remove this line